front/app/components/UI/MultipleSelect/index.tsx

```diff
@@ -91,8 +91,10 @@ export default class MultipleSelect extends React.PureComponent<Props, State> {
           options={options}
           onChange={this.handleOnChange}
           isDisabled={disabled}
-          styles={selectStyles}
+          styles={selectStyles()}
+          menuPosition="fixed"
```

Comment:
This makes sure all options are visible at all times. See Notion [ticket](https://www.notion.so/citizenlab/Multiselect-first-option-is-not-visible-nor-reachable-if-opened-upwards-on-phone-3f8970437a2e47bcb3e612a9fc928d4a?pvs=4) for current problem.

Reply:
As a safety check, you can try open some multi-select fields across the platform (on a phone). :-)